docs/src/pages/guides/migration-v4/migration-v4.md

```diff
@@ -1202,6 +1202,21 @@ As the core components use emotion as a styled engine, the props used by emotion
   +<Span>Create a user</Span>
   ```
 
+- Replace the `Typography` color values with values available in the [System](https://material-ui.com/system/palette/). We did this change to avoid collision of the component's vs the system props.
```

Comment:
Actually we should revert these changes if we still support the prev values...

Reply:
```suggestion
- Replace the `Typography` color values with values available in the [System](/system/palette/). We did this change to avoid collision of the component's vs the system props.
```

Reply:
I've changed the migration guide to just document the changes on the classes and style overrides and how those can be replaced. We can deprecate these values later and remove the transformation in v6.